### DISCUSSION TITLE
transfer 1.8.3 to 1.9.1

### DISCUSSION THREAD

**Discussion by DougBain** [mikeberger/borg_calendar#105] [Q&A] [answered]
I've used 1.8.3 (linux) happily for several years, transferring calendar data between laptop and PC by simply transferring the files borgdb.h2.db and borgdb.trace.db in .borg_db from one machine to the other.

I've now installed 1.9.1(amd64.deb) on a new PC and would like to transfer the old 1.8.3 calendar data onto it, but I see that the files in the 1.9.1 .borg_db are borgdb.trace.db and  a new file borg.mv.db.

What should I do? Can I just rename my old borgdb.h2.db to borg.mv.db and transfer the old files or what?

Would you please answer to douglas.bainbridge@cantab.net?
My old GitHub address is not working for e-mail.
Many thanks, Doug

**Comment by mikeberger** [comment, marked as answer]
The H2 database software that Borg uses changed its file format. The h2 and mv file formats are not compatible.

You should export your data from 1.8.3 and import into 1.9.1. You should upgrade to 1.9.1 on all machines if everything is working ok for you in 1.9.1.

You should be able to copy the borg.mv.db file between machines once you are on 1.9.1 everywhere. I have my database on Dropbox to avoid having to transfer files between machines.

BR,
Mike
